src/ralph/assets/tests/test_api.py

```diff
@@ -489,3 +531,21 @@ def test_tags(self):
         )
         response = self.client.get(url, format='json')
         self.assertEqual(len(response.data['results']), 1)
+
+    def test_str_field(self):
+        count = 0
+        for descendant in BaseObject._polymorphic_descendants:
+            if not descendant._polymorphic_descendants:
```

Comment:
There is `Asset` model, which inherits from `BaseObject`, but is not "final" type - `DataCenterAsset` and `BackOfficeAsset` are inheriting from it - so we're not exposing `assets` API endpoint and don't want to check it